packages/cli/src/commands/db.ts

```diff
@@ -9,7 +9,8 @@ const runPrisma = async (args: string[], silent = false) => {
   if (!prismaBin) {
     try {
       prismaBin = await getPrismaBin()
-    } catch {
+    } catch (err) {
+      log.error(err)
       throw new Error(
         "Oops, we can't find Prisma Client. Please make sure it's installed in your project",
       )
```

Comment:
Thanks! An even better solution is just to throw the original error which fixes the original issue. And this code is going to be removed soon anyways, so not too big of a deal.

```suggestion
      throw err
```